crates/emmylua_ls/src/handlers/initialized/client_config/mod.rs

```diff
@@ -30,8 +30,10 @@ pub async fn get_client_config(
     };
     match client_id {
         ClientId::VSCode => get_client_config_vscode(context, &mut config).await,
-        ClientId::Neovim => get_client_config_neovim(context, &mut config).await,
-        _ => Some(()),
+        ClientId::Neovim => {
+            get_client_config_default(context, &mut config, Some(&["Lua", "emmylua"])).await
```

Comment:
I think since LuaLS has already become the user preference for Neovim, there’s no need to request emmyLua anymore. Also, “Lua” is capitalized, while “emmylua” is lowercase, which doesn’t look as harmonious.

Reply:
I want to make it similar to VSCode. See PR description for detailed explanation:

> Currently, both LuaLs and EmmyLua use the same scope "Lua". It works in NeoVim because options for every LSP server are independent. In VSCode, however, they all share the same namespace, meaning that we can't reuse "Lua". I'd like to keep things consistent, hence this rename.